implementation/src/main/java/io/smallrye/openapi/runtime/scanner/OpenApiAnnotationScanner.java

```diff
@@ -449,6 +456,31 @@ private void processJaxRsResourceClass(OpenAPIImpl openApi, ClassInfo resourceCl
         }
     }
 
+    /**
+     * Build a map between exception class name and its corresponding @ApiResponse annotation in the jax-rs exception mapper
+     *
+     */
+    private Map<DotName, AnnotationInstance> processExceptionMappers() {
+        Map<DotName, AnnotationInstance> exceptionHandlerMap = new HashMap<>();
+        Collection<ClassInfo> exceptionMappers = this.index
+                .getKnownDirectImplementors(OpenApiConstants.DOTNAME_EXCEPTION_MAPPER);
+
+        for (ClassInfo classInfo : exceptionMappers) {
+            DotName exceptionDotName = classInfo.interfaceTypes().get(0).asParameterizedType().arguments().get(0).name();
```

Comment:
Is the interface type you want always going to be the one at index 0?

Reply:
@lamtrhieu - you can ensure the correct interface by doing something like this:

`classInfo.interfaceTypes().stream().filter(it -> it.name().equals(OpenApiConstants.DOTNAME_EXCEPTION_MAPPER)).findFirst().get()`

That will avoid issues where the exception mapper implements two or more interfaces and the first one is not the exception mapper.

Reply:
Nice - this is exactly what I was getting at.  +1

Reply:
@MikeEdgar I have modify the code with your approach. Please help to review again. 
Thanks